lib/nrf9160_time/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+zephyr_library()
+zephyr_library_sources(nrf9160_time.c)
+
+zephyr_include_directories(include)
```

Comment:
I believe it is best-practice for a library like this to place it's header file in

nrf/include/

instead of

nrf/lib/nrf9160_time/include

Among other things, it simplifies the compiler command.

Perhaps @rlubos  could confirm.

Reply:
I think this entry should just be removed, as there's no `nrf/lib/nrf9160_time/include` directory. But in general I agree, public APIs should end up in `nrf/include`

Reply:
Ah, I see that the header already was in nrf/include, then this is just some cargo cult code that we need to drop.

Reply:
Will change.